### PR TITLE
Add test file

### PR DESCRIPTION
**Pull request by nk18chi**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+test
+1
+2
+3
```
Comment (usk83) [inline test.txt]: English please

```
one
two
three
```

**Review by usk83 — CHANGES_REQUESTED** [review]
pls modify this part

**Review by usk83 — APPROVED** [review]
LGTM

**Comment by nk18chi** [comment]
This is closed because it's just for test.

**Pull request closed without merging**